### ISSUE TITLE
Slight bug with misnamed variable

### ISSUE THREAD

**Issue by StevePaget** [ankith26/Snake-python#1]
The variable "array" on line 12 should read "snake"

**Comment by ankith26** [comment]
Yes it should be snake. Originally all `snake` were `array`, then I figured out that I must come up with more creative variable names, so did the replacement everywhere but missed line 12.

Thanks for pointing it out.

**Comment by ankith26** [comment]
I have merged your PR.
